package.json

```diff
@@ -80,6 +80,7 @@
     "@elastic/charts": "^19.2.0",
     "babel-polyfill": "^6.26.0",
     "brace": "^0.11.1",
+    "elliptic": "^6.5.3",
```

Comment:
Is it necessary to add direct dependency on `elliptic`? In existing repo, elliptic^6.0.0 and ^6.5.2 is resolved to 6.5.2, can you only resolve it to 6.5.3 in `yarn.lock` instead of adding direct dependency?

Reply:
> Is it necessary to add direct dependency on `elliptic`? In existing repo, elliptic^6.0.0 and ^6.5.2 is resolved to 6.5.2, can you only resolve it to 6.5.3 in `yarn.lock` instead of adding direct dependency?

I considered that, although I just know by best practices that we are not supposed to edit `yarn.lock` directly. This was automatically populated in `package.json` when I ran `yarn upgrade elliptic@^6.5.3`. Let me know what you think.

Reply:
Cool, so I followed the suggestion [here](https://github.com/yarnpkg/yarn/issues/4986) to manually remove the dependency in `yarn.lock` and run `yarn` right after to re-populate the dependency with an upgraded version. There is still is an issue with yarn to be able to easily upgrade indirect/sub-dependencies, thanks for calling out.